src/uct/ib/base/ib_device.c

```diff
@@ -482,6 +482,40 @@ void uct_ib_handle_async_event(uct_ib_device_t *dev, uct_ib_async_event_t *event
     ucs_log(level, "IB Async event on %s: %s", uct_ib_device_name(dev), event_info);
 }
 
+static ucs_status_t uct_ib_device_get_sf_ids(uct_ib_device_t *dev)
```

Comment:
can we unite common code with uct_ib_device_get_sys_dev?

Reply:
Fixed.